config/application.rb

```diff
@@ -19,5 +19,8 @@ class Application < Rails::Application
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
     config.active_job.queue_adapter = :delayed_job
+
+    # Load modules in /lib
+    config.eager_load_paths += %W(#{config.root}/lib)
```

Comment:
I think rather than storing the module in lib and eager loading all of lib (which currently isn't much so it's not a big deal but does include the rake tasks), it may be better to create `app/lib` which is already autoloaded/eagerloaded depending on env and drop our modules in there. Happy to chat about this though.

Reply:
Yeah, that works for me. We could also make it a Concern, but I'm disinclined to do that since we don't need any of the Concern magic.